server/container_remove.go

```diff
@@ -42,7 +42,12 @@ func (s *Server) RemoveContainer(ctx context.Context, req *types.RemoveContainer
 			}
 		}
 	}
-
+	if s.config.EventedPLEG {
+		err := s.generateCRIEvent(ctx, c, types.ContainerEventType_CONTAINER_DELETED_EVENT)
+		if err != nil {
+			log.Errorf(ctx, "Unable to generate event %s for container %s due to err %s", types.ContainerEventType_CONTAINER_DELETED_EVENT, c.ID(), err)
```

Comment:
If we just log the errors, why do we even bother returning it from `generateCRIEvent` ? We could combine the error logging into the method to reduce the overhead. We can also check `if s.config.EventedPLEG` in `generateCRIEvent` to make it a no op.

Reply:
Done